scene/main/scene_tree.cpp

```diff
@@ -1400,6 +1401,29 @@ SceneTree::SceneTree() {
 	bool snap_2d_vertices = GLOBAL_DEF("rendering/2d/snap/snap_2d_vertices_to_pixel", false);
 	root->set_snap_2d_vertices_to_pixel(snap_2d_vertices);
 
+	// We setup VRS for the main viewport here, in the editor this will have little effect.
```

Comment:
I think its kind of pointless to expose this here, why would users want to set a VRS texture or modes for the main game? I would leave it in Viewport but not expose those.

Reply:
Well for XR you'd want it active at startup so this is just to set that up so you don't need to write code for it. 

For normal games you likely won't enable it until you're in the game itself so I guess there you'd have to do it in code anyways.

Reply:
It's true that it's not easy to configure the `/root` main viewport as it's not part of the main scene so you can't set its properties in the Inspector. So it makes sense IMO to have settings exposed for it so that it can be configured without having to do it e.g. in the `_ready` of the main scene's root script (provided that happens early enough).